variants/RUMBA32_F446VE/variant.h

```diff
@@ -27,14 +27,14 @@ extern "C" {
  *        Pins
  *----------------------------------------------------------------------------*/
 
-#define PA0   0  //D0
-#define PA1   1  //D1
-#define PA2   2  //D2
-#define PA3   A0 //D3
-#define PA4   A1 //D4
-#define PA5   5  //D5
-#define PA6   6  //D6
-#define PA7   7  //D7
+#define PA0   0  //D0   //A7
+#define PA1   1  //D1   //A8
+#define PA2   2  //D2   //A9
+#define PA3   3  //D3   //A0
+#define PA4   4  //D4   //A1
+#define PA5   5  //D5   //A10
+#define PA6   6  //D6   //A11
+#define PA7   7  //D7   //A12
```

Comment:
You have to use Ax value else if you use the analogRead API with PYn this will not work.
```suggestion
#define PA0   A7  //D0
#define PA1   A8  //D1
#define PA2   A9  //D2
#define PA3   A0  //D3
#define PA4   A1  //D4
#define PA5   A10  //D5 
#define PA6   A11  //D6
#define PA7   A12  //D7
```
same for all other...

Reply:
Thanks @fpistm, not sure what I was thinking there... 

Fixed and tested AnalogRead using both Ax and PYn naming. Results look correct (same readings for either naming). Gave this a quick test and results are consistent either way:

![ADCtest](https://user-images.githubusercontent.com/5538631/84139809-b3b12e00-aa8f-11ea-8a60-b1b7c5aa152c.png)